pkg/nsx/services/inventory/build.go

```diff
@@ -127,6 +127,16 @@ func (s *InventoryService) BuildIngress(ingress *networkingv1.Ingress) (retry bo
 		preIngress = *preIngress.(*containerinventory.ContainerIngressPolicy)
 	}
 
+	// Get network errors from ingress annotations
+	var networkErrors []common.NetworkError
+	for key, value := range ingress.Annotations {
+		if key == "ncp/error.loadbalancer" {
+			networkErrors = append(networkErrors, common.NetworkError{
+				ErrorMessage: key + ":" + value,
+			})
+		}
+	}
+
 	containerIngress := containerinventory.ContainerIngressPolicy{
```

Comment:
could you also update the NetworkStatus? and update the networkErrors based on NetworkStatus?

Reply:
fixed